README.md

```diff
@@ -56,9 +62,12 @@ The result of comparing `$new` with `$old` will return a new array with the chan
 **Comparisons are strict by default**, but you can specify that you want to make a loose comparison passing a boolean as the third parameter for `compare` method or calling the `looseComparison`
 
 ```php
-// This will deactivate the strict comparison mode
+// Passing 'false' as a thid parameter will deactivate the strict comparison mode
```

Comment:
```suggestion
// Passing 'false' as a third parameter will deactivate the strict comparison mode
```

Reply:
Thank you! 
I should install a spelling checker on my IDE

Reply:
This one ;)
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker